app/models/chargeback_rate_detail.rb

```diff
@@ -90,4 +90,8 @@ def rate_type
     # Return parent's rate type
     chargeback_rate.rate_type unless chargeback_rate.nil?
   end
+
+  def detail_currency
```

Comment:
Why not make this an ActiveRecord relation instead of a method? Something like this:
```
belongs_to :detail_curreny, :class_name => "ChargebackRateDetailCurrency", :foreign_key => :chargeback_rate_detail_currency_id
```